server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/K8sTaskScheduler.java

```diff
@@ -116,6 +116,24 @@ public void stop(Collection<Task> tasks) {
         });
     }
 
+    @Override
+    public String[] exec(Task task, String... command) {
```

Comment:
How about return a Future object , since the `k8sClient.execInPod` is a thread blocking action. Returnning Future object could let people who is using this interface know that the `exec` is a time consuming operation. The tomcat container has thread number limit , abusing this method could cause server panic.

Reply:
Done